test/test.tf

```diff
@@ -33,7 +33,7 @@ module "test_nodes_sydney" {
 }
 
 module "test_nodes_sydney_channels" {
-  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=master"
+  source            = "github.com/aeternity/terraform-aws-aenode-deploy?ref=test-on-multiple-terraform-versions"
```

Comment:
will be fixed after https://github.com/aeternity/terraform-aws-aenode-deploy/pull/31 merge as master have issues